plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/controller/CodeTransformChatController.kt

```diff
@@ -90,7 +92,6 @@ class CodeTransformChatController(
         val validationResult = codeModernizerManager.validate(context.project)
 
         if (!validationResult.valid) {
-            codeModernizerManager.warnUnsupportedProject(validationResult.invalidReason)
```

Comment:
Pre-existing issue which was leading to the chat freezing when you select No JDK under Project Structure settings. We don't need this extra notification anyway, since we show the user the error message through the chat right below too, so just deleted it.